src/test/groovy/de/fntsoftware/gradle/MarkdownToPdfPluginTest.groovy

```diff
@@ -45,6 +48,8 @@ class MarkdownToPdfPluginTest {
 			MarkdownToPdfTask task = project.tasks.create("myTask", MarkdownToPdfTask)
 			task.inputFile = getClass().getResource("READMETest.md").toURI()
 			task.outputFile = project.projectDir.path + '/READMETest.pdf'
+			task.setOption(Parser.EXTENSIONS, Arrays.asList(TablesExtension.create()))
```

Comment:
Can you please create a new Test for it ?

Reply:
I created a separate test for it.  If it is not what you were looking for, just let me know and I can make a more specific/better test.